### ISSUE TITLE
[dev.icinga.com #8353] Group related links using the nav and ul elements

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#1360]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/8353

**Created by elippmann on 2015-02-04 11:40:06 +00:00**

Assignee: _elippmann_
Status: _Closed_  (closed on _2016-02-19 16:07:05 +00:00_)
Target Version: _Accessibility_
Last Update: _2016-02-19 16:07:05 +00:00 (in Redmine)_

---

http://www.w3.org/TR/2008/WD-WCAG20-TECHS-20080430/H50.html wrote:
> The objective of this technique is to demonstrate how to group links into logical sets. When links are grouped into logical sets (for example, in a navigation bar or main menu that appears on every page in a site) they should be marked up as a unit. Navigation bars are usually the first thing someone encounters on a page. People who are sighted are often able to ignore navigation parts and start reading the content of the page. Someone using a screen reader must first listen to the text of each link in the navigation bar before reading the interesting content. There are several ways to mark up content so that a user with a screen reader can jump over the navigation bar and avoid reading all of the links.

http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140916/H97 wrote:
> The objective of this technique is to group navigation links using the HTML5 nav element. The nav element is one of several sectioning elements in HTML5. Use of this markup can make groups of links easier to locate and skip past by users of assistive technology such as screen readers. Using semantic structures allow custom style sheets to be used to change the presentation of groups of links while preserving their relationship. When the nav element is employed more than once on a page, distinguish the navigation groups by using an aria-label or aria-labelledby attribute. \[...\]

We should group links using <code>nav</code> and <code>ul</code>. This applies to links in our state summaries, tabs and our menu, e.g.:

    Services Pagination

        
            Page 1
            Page 2
        

For the links for modifying how many rows to display on one page and the pagination it's sufficient to group these links using the <code>ul</code> element because these links do not represent a discrete section of the page.

Note that adding ARIA properties or landmarks to grouped links and adding markup for jumping to the links is not part of this issue.

---

**Relations**:

* blocks [#8351](https://dev.icinga.com/issues/8351)
